web-console/Storage/OCS/OCS _Installation_Workflow.md

```diff
@@ -0,0 +1,119 @@
+# OCS Install Workflow
+
+### OperatorHub
+
+There will be 2 OCS operator entries in OperatorHub covering each of the following modes. 
+* Converged Mode -- Local Cluster / Hyperconverged 
+* Independent Mode -- External Cluster (Ceph Cluster is deployed externally)
+
+### Converged Mode Design
+
+OperatorHub
+![OperatorHub operators view](img/OCS-operator-view.png)
+
+Installation
+![OperatorHub operators view](img/OCS-Install-step1.png)
+
+![OperatorHub operators view](img/OCS-Install-step2.png)
+
+After the subscription process, the user will see the subscribed operator page. 
+
+![subscription operators page](img/OCS_Subscription_page.png) 
+
+In the installed operator view the user can see the OCS operator installed. 
+
+![OperatorHub operators view](img/Installed_operators_list.png)
+
+
+# The OCS installation process
+OCS Overview page
+![OCS-View](img/OCS-View.png)
+
+Clicking on "create new" (OCS Service) Will start the following installation flow: 
+
+Based on the underlying platform, the user can select between “Create new nodes” (IPI) and “Use existing nodes” (UPI).
+* Create new nodes -- System (OCS) creates new nodes along with underlying storage devices/volumes in the cloud, i.e. AWS, Azure, Google
+    * Note: not applicable for VMware currently and not applicable for baremetal
+* Use existing nodes -- uses existing OCP nodes and existing storage devices/volumes (in AWS and VMware, and for baremetal), which Admin has to select
+
+
+
+## For AWS or Cloud Deployments (IPI and UPI)
+* “Create New Nodes” (IPI) option:
+In AWS, three m4.large EC2 nodes will be created with the appropriate storage node label, which would be used by OCS.
+
+A new bucket will be created automatically in the AWS S3 connected cloud, in the same region as the OpenShift.
+
+![create new](img/Create_new_OCS_00.png)
```

Comment:
The text underneath the title "OCS runs as a cloud-native service..." is missing some text, i.e. "behind the scenes tasks such as..."

@shirimordechay